bindings/GstApp/pkg.info

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "gi-gstapp",
+    "version": "1.0.28",
+    "description": "Bindings for GStreamerApp, autogenerated by haskell-gi.",
+    "synopsis": "GStreamerApp bindings",
+    "author": null,
+    "girName": "GstApp",
+    "girVersion": "1.0",
+    "girOverrides": "GstApp.overrides",
```

Comment:
I guess I also need to remove this? Should I set it to `null` or something?

Reply:
Yes, that's right, you can set `girOverrides` to `null` to indicate there's no overrides file. Thanks again for the work on the PR!

Reply:
Done!